### DISCUSSION TITLE
new webpack.AutomaticPrefetchPlugin() cause error after 5.77.0

### DISCUSSION THREAD

**Discussion by passariello** [webpack/webpack#17235] [Q&A]
# Bug report

**What is the current behavior?**

Use new webpack.AutomaticPrefetchPlugin()

**If the current behavior is a bug, please provide the steps to reproduce.**

Add new webpack.AutomaticPrefetchPlugin() to config and run

**What is the expected behavior?**

Normal operation

**Other relevant information:**
webpack version:  >5.77
Node.js version: latest
Operating System: windows 11
Additional tools:

**Comment by alexander-akait** [comment]
> Add new webpack.AutomaticPrefetchPlugin() to config and run

Sorry, can't reproduce and we have tests with this plugin, please create reproducible test repo and I will reopen, thank you

**Comment by passariello** [comment]
[webpack_issue.zip](https://github.com/webpack/webpack/files/11522998/webpack_issue.zip)

So, I create this repo from existing project... It's not run as APP but console show the issue for you.
If you run it and watch console you can see general errors about Json files (also from node_modules) ... 
if you comment new webpack.AutomaticPrefetchPlugin() the issue go away... it's something connected with copyPlugin too

Sorry for the repo... I can't share the entire project and I can't reproduce in other projects due "company or proprietary" code.
It's the best that I can... also, this ticket is connected with the previous, probably the previous can be deleted/

Thanks for your effort. It's not a stopper for me but I want to help you guys because I love webpack :)

**Comment by alexander-akait** [comment]
@passariello Sorry I can't use your repo due `"nrg-common": "bitbucket:tymeware/nrg-common",`

**Comment by alexander-akait** [comment]
I moved it to dicussions when we can reproduce it I will reopen or fix it, also please provide an error stack/message/etc

**Comment by passariello** [comment]
[webpack_issue_1.zip](https://github.com/webpack/webpack/files/11525841/webpack_issue_1.zip)

So sorry, I completely forgot the custom packages... I remove it.
Still the issue for your convenience.

PS: I have replicate it onto another project,,, exactly the same issue after 5.77.0.
Before 5.78.0 I have no issue if helps!

Thanks Alex!
